cargo-ledger/src/main.rs

```diff
@@ -194,13 +194,34 @@ fn build_app(
 ) {
     let exe_path = match use_prebuilt {
         None => {
+            let c_sdk_path = match device {
+                Device::Nanos => std::env::var("NANOS_SDK"),
+                Device::Nanosplus => std::env::var("NANOSP_SDK"),
+                Device::Nanox => std::env::var("NANOX_SDK"),
+            };
+
+            let mut args: Vec<String> = vec![];
+            args.push(String::from("build"));
+            args.push(String::from("--release"));
+            args.push(format!("--target={}", device.as_ref()));
+            args.push(String::from(
+                "--message-format=json-diagnostic-rendered-ansi",
+            ));
+
+            match std::env::var("LEDGER_SDK_PATH") {
+                Ok(_) => (),
+                Err(_) => {
+                    if let Ok(path) = c_sdk_path {
+                        args.push(format!(
+                            "--config env.LEDGER_SDK_PATH={}",
+                            path
+                        ))
+                    }
```

Comment:
Should we do something when `c_sdk_path` is an `Err()` too? Abort?

Reply:
Yes in that case, C SDK will be have to be cloned when building sys crate